src/rpm/builder.rs

```diff
@@ -217,14 +201,38 @@ impl RPMBuilder {
         self
     }
 
-    pub fn add_changelog_entry<E, F>(mut self, author: E, entry: F, time: u32) -> Self
+    /// Add an entry to the changelog, compromised of author, description, and the date and time of the change.
+    ///
+    /// ```ignore
+    /// let pkg = rpm::RPMBuilder::new(..)
+    ///     .add_changelog_entry(
+    ///         "Alfred J. Quack <quack@example.com>",
+    ///         r#" - Obsolete `fn foo`, in favor of `fn bar`.
+    /// - Secondly."#,
+    ///         rpm::chrono::Utc.timestamp_opt(1681411811, 0).unwrap(),
+    ///     )
+    ///     .add_changelog_entry(
+    ///         "Gambl B. Xen <gbx@example.com>",
+    ///         " - Add enumerator."
+    ///         rpm::chrono::DateTime::parse_from_rfc3339("1996-12-19T16:39:57-08:00").unwrap(),
+    ///     )
+    ///     //..
+    /// ```
+    pub fn add_changelog_entry<A, E, TZ>(
+        mut self,
+        author: A,
```

Comment:
e.g. you could say something along the lines of

> Convention is to have one changelog entry per release.  That the changelog name should be comprised of the first and last name of the release publisher, followed by their email address inside angle brackets, followed by a hyphen separator, followed by the version identifier of the release.  Changes within the release should be listed in a bulleted list, one-per-line, using a hyphen character as a bullet.

Reply:
And rename the parameter

```suggestion
        changelogname: N,
```